Classes/Importer/AbstractImporter.php

```diff
@@ -551,27 +601,46 @@ protected function getLabelFromRecordData(array $data)
      *
      * The storage node is either created or just retrieved and finally stored in $this->storageNode.
      *
-     * @param string $nodePath Absolute or relative (to the site node) node path of the storage node
+     * @param string $nodePathOrIdentifier Absolute or relative (to the site node) node path of the storage node
      * @param string $title Title for the storage node document
      * @return void
      * @throws \Neos\ContentRepository\Exception\NodeTypeNotFoundException
      */
-    protected function initializeStorageNode($nodePath, $title)
+    protected function initializeStorageNode($nodePathOrIdentifier, $title)
+    {
+        if (is_string($nodePathOrIdentifier) && $nodePathOrIdentifier[0] === '#') {
```

Comment:
Yes, but I think if you decide to import under a specific node (by identifier) you don't want the magic of auto created storage node, no ?